phoenix-core/src/main/java/org/apache/phoenix/schema/types/PhoenixArray.java

```diff
@@ -561,6 +666,23 @@ public int estimateByteSize(int pos) {
 		}
 		
 		@Override
+		public String toString() {
+			StringBuilder sb = new StringBuilder(TO_STRING_BEGIN);
+			boolean isFirst = true;
+			final Format defaultFormat = null;
+			for (Byte o : byteArr) {
+				if (isFirst) {
+					isFirst = false;
+				} else {
+					sb.append(TO_STRING_SEPARATOR);
+				}
+				sb.append(this.baseType.toStringLiteral(o, defaultFormat));
```

Comment:
Use these methods instead of relying on unboxing:

    PhoenixArray a;
    for (int i = 0; i < a.getDimensions(); i++) {
      Object o = a.getElement(i);
    }